feature/news/src/main/java/com/teamwable/news/NewsFragment.kt

```diff
@@ -59,11 +108,13 @@ class NewsFragment : BindingFragment<FragmentNewsBinding>(FragmentNewsBinding::i
                     NewsTabType.NEWS.idx -> {
                         trackEvent(CLICK_NEWS)
                         setBadgeOnNews(NewsTabType.NEWS.idx, false)
+                        lifecycleScope.launch { viewModel.saveNewsNumber(serverNewsNumber) }
```

Comment:
p1 : 프레그먼트 생명주기 내에서 실행되도록 수정해주세요!
```suggestion
                        viewLifecycleScope.launch { viewModel.saveNewsNumber(serverNewsNumber) }
```

Reply:
뒤죽박죽